modules/vpc-sc/main.tf

```diff
@@ -89,6 +93,93 @@ resource "google_access_context_manager_service_perimeter" "standard" {
           allowed_services   = each.value.enforced_config.vpc_accessible_services
         }
       }
+
+      dynamic "egress_policies" {
+        for_each = try(local.perimeter_egress_policies_enforced[each.key] != null ? local.perimeter_egress_policies_enforced[each.key] : [], [])
+
+        content {
+          dynamic "egress_from" {
+            for_each = try(var.egress_policies[egress_policies.value].egress_from != null ? [""] : [], [])
+
+            content {
+              identity_type = try(var.egress_policies[egress_policies.value].egress_from.identity_type, null)
+              identities    = try(var.egress_policies[egress_policies.value].egress_from.identities, null)
+            }
+          }
+          dynamic "egress_to" {
```

Comment:
Likewise, maybe skip the dynamic block.

Reply:
Those are not required over API, I did a test and it works. I'll keep it, we can change the behaviour in a future iteration